packages/dds/map/src/mapKernel.ts

```diff
@@ -342,6 +342,9 @@ export class MapKernel {
 		// Clear the data locally first.
 		this.clearCore(true);
 
+		// Clear the pendingKeys immediately, the local unack'd operations are aborted
+		this.pendingKeys.clear();
```

Comment:
seems reasonable, but I think you also need to update logic in resubmit--specifically I think we're now in danger of hitting 0x2ff.